generators/spa/templates/Vue3JS/Module.dnn

```diff
@@ -0,0 +1,101 @@
+<dotnetnuke type="Package" version="5.0">
+  <packages>
+    <package name="<%= fullNamespace %>" type="Module" version="01.00.00">
+      <friendlyName><%= friendlyName %></friendlyName>
+      <description><![CDATA[<p><%= extensionDescription %></p>]]></description>
+      <iconFile>DesktopModules/<%= friendlyName %>/Images/logo.png</iconFile>
+      <owner>
+        <name><%= ownerName %></name>
+        <organization><%= companyName %></organization>
+        <url><![CDATA[<a href="<%= companyUrl %>" target="_blank"><%= companyUrl %></a>]]></url>
+        <email><![CDATA[<a href="mailto:<%= emailAddress %>" target="_blank"><%= emailAddress %></a>]]></email>
+      </owner>
+      <license src="License.txt"></license>
+      <releaseNotes src="ReleaseNotes.txt"></releaseNotes>
+      <dependencies>
+        <dependency type="CoreVersion">09.05.01</dependency>
```

Comment:
@alejoroman0605 The DNN version number is wrong. It should be 9.8.0.

Reply:
@WillStrohl I just fixed it https://github.com/UpendoVentures/generator-upendodnn/pull/280/commits/c0437c2115848e91e47ac709f478d2f74ff60337